params/config.go

```diff
@@ -505,7 +505,7 @@ func (c *ChainConfig) IsCancun(num *big.Int, time uint64) bool {
 
 // IsPrague returns whether num is either equal to the Prague fork time or greater.
 func (c *ChainConfig) IsPrague(num *big.Int, time uint64) bool {
-	return c.IsLondon(num) && isTimestampForked(c.PragueTime, time)
+	return c.IsShanghai(num, time) && isTimestampForked(c.PragueTime, time)
```

Comment:
I think we might need to do a similar change in `kaustinen-with-shapella` since I Think using `IsLondon` isn't entirely correct?